syft/generic/pointers/pointer_plan.py

```diff
@@ -43,18 +47,53 @@ def __init__(
             description=description,
         )
 
+    # Make PointerPlan compatible with multi pointers
+    @property
+    def location(self):
+        n_locations = len(self._locations)
+        if n_locations != 1:
+            return self._locations
+        else:
+            return self._locations[0]
+
+    @location.setter
+    def location(self, new_location: Union[AbstractWorker, List[AbstractWorker]]):
+        if isinstance(new_location, AbstractWorker):
+            self._locations = [new_location]
+        else:
+            self._locations = new_location
+
+    @property
+    def id_at_location(self):
+        n_ids = len(self._ids_at_location)
+        if n_ids != 1:
+            return self._ids_at_location
+        else:
+            return self._ids_at_location[0]
+
+    @id_at_location.setter
+    def id_at_location(self, new_id_at_location):
+        if isinstance(new_id_at_location, (int, str)):
```

Comment:
since we're already doing a type check, could we throw an error if it is not an accepted type?

Reply:
I'll change the type check to be simply against the args being a list/tuple or not, it will prevent making assumption about the format of an id